http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1344,6 +1314,19 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
     }
   }
 
+  // This should probably be moved to WsmDao, if HttpWsmDao switches to using the WSM api clients
```

Comment:
What do you mean by that?

Reply:
At current, `HttpWsmDao` doesn't use the swagger-generated WSM clients. It's building paths directly with strings instead. This class already uses the `ResourceApi` though, so it seemed better to continue using the generated client directly, even if it wasn't in the dao. 

If the dao were to start using the published WSM clients, that would be the place where I'd expect this to live, since it is a thin wrapper around the api, with some common error handling.